diff --git a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
index b6fbc7a..5f5d04c 100644
--- a/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
+++ b/node_modules/gatsby-theme-portfolio-minimal/src/components/Project/index.tsx
@@ -64,7 +64,6 @@ export function Project(props: ProjectProps): React.ReactElement {
                                     key={key}
                                     href={link.url}
                                     target="_blank"
-                                    rel="nofollow noopener noreferrer"
                                     aria-label="External Link"
                                 >
                                     <Icon name={link.type} color="var(--subtext-color)" />
@@ -74,7 +73,7 @@ export function Project(props: ProjectProps): React.ReactElement {
                 </div>
             </div>
             {props.data.image.src && props.data.image.linkTo && (
-                <a href={props.data.image.linkTo} target="_blank" rel="noopener noreferrer" aria-label="External Link">
+                <a href={props.data.image.linkTo} target="_blank" aria-label="External Link">
                     <GatsbyImage
                         className={classes.ProjectImageWrapper}
                         imgClassName={classes.ProjectImage}
